hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/StateLifecyclesTest.java

```diff
@@ -129,7 +129,7 @@ private AddressBook createPretendBookFrom(final Platform platform, final boolean
         }
         final var nodeId = platform.getSelfId().getIdAsInt();
         final var address = new Address(
-                nodeId,
+                new NodeId(nodeId),
```

Comment:
Rather than calling `getIdAsInt` and constructing a new `NodeId` with the int value, you can simply save the `NodeId` returned by `getSelfId`, to pass that into the `Address` constructor